bedrock/base/templates/base-pebbles.html

```diff
@@ -84,7 +84,6 @@
       data-global-close="{{ _('Close') }}"
       data-global-next="{{ _('Next') }}"
       data-global-previous="{{ _('Previous') }}"
-      data-global-update-firefox="{{ _('Update your Firefox') }}"
```

Comment:
Note: this global string doesn't seem to be used anywhere as far as I can tell. I'm removing it here to try and keep what's in `download_button.lang` specific to download buttons, as opposed to being a common source for general purpose strings.